modules/react/common/stories/BasicCompoundComponent.stories.mdx

```diff
@@ -0,0 +1,286 @@
+    menuConfig: {} as typeof useMenuModel.TConfig,
+  },
+  requiredConfig: useOverflowListModel.requiredConfig,
+})(config => {
+  // define your internal state and events
+});
+```
+
+### getElemProps
+
+This function will separate all `elemProps` or default attributes from an element from the model
+config props. If a prop is both a `config` _and_ an `elemProp`, you can manually apply the prop
+again. `elemProps` is called internally automatically by createContainer. If you use
+`createContainer`, you shouldn't need to use this function.
+
+By default `createModelHook` does this for you and spread the `elemProps` onto the component.
+
+# createContainer
+
+When building a component that has a model, you typically start with a container component.
+`createContainer` is a utility function that hooks up all the pieces for you.
+
+## Usage
+
+- It will wrap your component in a context provider so that `subComponents` have access to the
+  model.
+- It attaches your model hook to your component
+- It runs any `elemPropsHook` hooks defined
+- You can attach `subComponents`.
+- It will also extract the element attributes and define the `ref` type based on the element you
+  give it.
+
+```tsx
+import {createContainer} from '@workday/canvas-kit-react/common';
+interface DisclosureProps {}
+
+//... useDiscloseModel
+
+export const Disclosure = createContainer('div')({
+  displayName: 'Disclosure',
+  modelHook: useDisclosureModel,
+})<DisclosureProps>(({children, ...elemProps}, Element, model) => {
+  return (
+    // spread div attributes including ref
+    <Element {...elemProps}>{children}</Element>
+  );
+});
+```
+
+<ExampleCodeBlock code={CreateContainer} />
+
+# createSubcomponent
+
+Once you've built a container component, adding subcomponent is easy. `createSubcomponent` is
+similar to `createContainer` in that it hooks up a lot of the piece for you. The main difference
+being that is uses the context created by `createContainer`. This allows access to the model created
+at the root level so that you can use any state or events at a child component level.
+
+## Usage
+
+- It subscribes to the parent context allowing you access to the model hook `state` and `events`
+- It attaches your model hook to your component
+- It runs any `elemPropsHook` hooks defined
+- You can attach `subComponents`.
+- It will also extract the element attributes and define the `ref` type based on the element you
+  give it.
+
+```tsx
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+export interface DisclosreTargetProps {
+  /**
+   * The children of the `Expandable.Target`
+   */
+  children?: React.ReactNode;
+}
+
+//... useDiscloseModel
+
+export const DisclosureTarget = createSubcomponent('button')({
+  modelHook: useDisclosureModel,
+})<DisclosreTargetProps>(({children, ...elemenProps}, Element, model) => {
```

Comment:
Wasn't sure if this was supposed to be `elemProps` or `elementProps`. You used `elementProps` in a another code snippet in the document so sticking with that here.
```suggestion
})<DisclosureTargetProps>(({children, ...elementProps}, Element, model) => {
```